checker/tests/index/MinLenFromPositive.java

```diff
@@ -0,0 +1,21 @@
+import org.checkerframework.checker.index.qual.Positive;
+import org.checkerframework.common.value.qual.*;
+
+class MinLenFromPositive {
+
+    void test(@Positive int x) {
+        int @MinLen(1) [] y = new int[x];
+        @IntRange(from = 1) int z = x;
+        @Positive int q = x;
+    }
+
+    @SuppressWarnings("index")
```

Comment:
I did this, but I can't seem to get `ant value-tests` to run this test automatically (since it's not in the same directory?). Is there some way to get that test to run automatically whenever `ant value-tests` is run?